[BD-5052]:add a warning for "Unsubscribe from specific subscription group" (#10268)
braze-inc/braze-docs: _docs/_user_guide/administrative/app_settings/email_settings.md

- **Unsubscribe globally from all emails**: Uses the Braze default one-click unsubscribe header. Users who click the unsubscribe button will have their global email subscription state set to "Unsubscribed".
- **Unsubscribe from specific subscription group**: Uses the specified subscription group. Users who click the unsubscribe button will be unsubscribed from the selected subscription group.
    - When selecting a subscription group, add the **Subscription Group** filter in **Target Audiences** to only target users who are subscribed to this specific group. The subscription group selected for one-click unsubscribe must match the subscription group you’re targeting. If there is a mismatch in the subscription group, you may risk sending to a user who is trying to unsubscribe from a subscription group they're already unsubscribed from.

> {% alert note %}
> Please note that this setting only applies to the one-click list-unsubscribe header. The mailto list-unsubscribe header is not affected when selecting this option. This means a recipient that unsubscribes via this method will log a global unsubscribe, not an unsubscribe from the specific subscription group. If you'd like to exclude the mailto list-unsubscribe header from globally unsubscribing users, when selecting this setting, please reach out to Support.
> {% endalert %}

Reviewer: ```suggestion

{% alert important %}
This setting only applies to the one-click list-unsubscribe header. The mailto list-unsubscribe header is not affected when selecting this option. This means a recipient who unsubscribes using this method will log a global unsubscribe, not an unsubscribe from the specific subscription group. To exclude the mailto list-unsubscribe header from globally unsubscribing users, when selecting this setting, contact [Support].
{% endalert %}

```
